docs/sources/reference/commandline/cli.md

```diff
@@ -1410,7 +1410,7 @@ for further details.
 
     rename a existing container to a NEW_NAME
 
-The `docker rename` command allows the container to be renamed to a different name.  
```

Comment:
Sure, no problem. I think my editor removed all the spaces at the end of those lines.